src/it/scala/org/aerospiker/OperationSpec.scala

```diff
@@ -367,6 +363,27 @@ class OperationSpec extends FlatSpec with Matchers with BeforeAndAfter {
 
   }
 
+  it should "scan all" in {
+
+    import policy.Implicits._
+
+    val c = Client(hosts)
+
+    try {
+      val f = c.scanAll("test", "teste")
+      val r = f.runFor(3000)
+
+      println(r.size)
+
+      assert(r.find(_.isLeft).isEmpty)
+
+    } catch {
+      case e: Throwable=> fail(e)
```

Comment:
You don't need to explicitly call `fail` as the spec fails in case exception was thrown inside a spec.